src/modules/dashboard/components/ExpenditurePage/Batch/DownloadTemplate/DownloadTemplate.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useMemo } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import ExternalLink from '~core/ExternalLink';
+
+import styles from './DownloadTemplate.css';
+
+export const MSG = defineMessages({
+  downloadTemplate: {
+    id: 'dashboard.ExpenditurePage.Batch.downloadTemplate',
```

Comment:
This id should be `dashboard.ExpenditurePage.Batch.DownloadTemplate.downloadTemplate`

If you don't want to repeat the "downloadTemplate" then you could name it something like `dashboard.ExpenditurePage.Batch.DownloadTemplate.linkText`